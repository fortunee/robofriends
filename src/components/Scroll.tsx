import React from 'react';

const Scroll: React.FC = ({ children }) => (
  <div style={{ overflowY: 'scroll', height: '80vh'}}>
    {children}
  </div>
)

export default Scroll;
