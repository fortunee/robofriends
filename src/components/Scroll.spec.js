import { shallow } from 'enzyme';
import React from 'react';

import Scroll from './Scroll';

it('Should render a Scroll component', () => {
  expect(shallow(<Scroll />).debug()).toMatchSnapshot();
});
