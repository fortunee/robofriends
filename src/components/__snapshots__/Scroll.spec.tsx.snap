// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render a Scroll component 1`] = `"<div style={{...}} />"`;
